browser/css/jsdialogs.css

```diff
@@ -182,6 +182,10 @@ td.jsdialog > [id^='table-box']:not(.sidebar) {
 	grid-gap: 4px;
 }
 
+#viewgrid {
```

Comment:
please add more precise selector, so it will also contain dialog's id (names could be repeated somewhere else)

Reply:
Thanks! updated the selector to include the dialog's Id.